### DISCUSSION TITLE
Language choice

### DISCUSSION THREAD

**Discussion by mlana** [Cookie-AutoDelete/Cookie-AutoDelete#975] [Q&A / Support] [answered]
hi, i would like to know how to choose the language* of the interface of CAD.
i am in Itaoy but i don't like so much the itaoian translation, i wold prefere CAD to speak to me in English.
thanks
maurizio

*i searched for "language" in the forums but i found nothing

**Comment by kennethtran93** [comment, marked as answer]
Make sure to also check the 'closed' issues as well.  This is duplicate of #755.

You will need to change the browser language itself to get the translations.  Refer to your browser help to see how to change it.  We simply (in our code) rely on `browser.i18n.getMessage` to get the translated message, so we leave it to the browser for the language to translate to.

**Comment by kennethtran93** [comment]
If you would like to help translate the strings, feel free to look at our CrowdIn page to get started.
